pyi.py

```diff
@@ -354,6 +366,7 @@ def should_warn(self, code):
 Y008 = 'Y008 Unrecognized platform "{platform}"'
 Y009 = 'Y009 Empty body should contain "...", not "pass"'
 Y010 = 'Y010 Function body must contain only "..."'
+Y011 = 'Y011 Default values for typed arguments must be "..."'
```

Comment:
Put it also in the README.

Reply:
Done. Also rebased the branch because I made it on top of an old commit before we added docs to README.